### PR TITLE
Publish OSLC AM v3.0 PSD01

### PR DESCRIPTION
**Pull request by jamsden**

### CODE REVIEW

File: specs/am/architecture-management-spec.html [hunk 1]
```diff
@@ -59,6 +59,18 @@
             title: "OSLC Architecture Management Version 3.0. Part 2: Vocabulary",
             href: "architecture-management-vocab.html",
           },
+          {
+            title: "OSLC Architecture Management Version 3.0. Part 3: Constraints",
+            href: "architecture-management-shapes.html",
+          },
+          {
+            title: "OSLC Architecture Management Version 3.0. Part 4: Machine Readable Vocabulary Terms",
```
Comment (berezovskyi) [inline specs/am/architecture-management-spec.html:67]: Part needs to be removed from here. Could you perhaps add it to the checklist too, please?

File: specs/am/architecture-management-spec.html [hunk 2]
```diff
@@ -59,6 +59,18 @@
...
+            href: "architecture-management-vocab.ttl",
+          },
+          {
+            title: "OSLC Architecture Management Version 3.0. Part 5: Machine Readable Constraints",
```
Comment (berezovskyi) [inline specs/am/architecture-management-spec.html:71]: "Part" needs to be removed from here. Could you perhaps add it to the checklist too, please?

File: specs/am/architecture-management-spec.html [hunk 3]
```diff
@@ -800,7 +812,7 @@ <h2>Samples</h2>
       <p>
         See
         <a
-          href="http://open-services.net/wiki/architecture-management/OSLC-Architecture-Management-2.0-Appendix-A%3A-Samples"
+          href="https://archive.open-services.net/wiki/architecture-management/OSLC-Architecture-Management-2.0-Appendix-A_-Samples/index.html"
```
Comment (berezovskyi) [inline specs/am/architecture-management-spec.html:815]: Maybe let's leave the original URL? Otherwise be very hard to change redirects later.

File: specs/cm/change-mgt-shapes.html [hunk 4]
```diff
@@ -2,30 +2,29 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>OSLC Change Management Version 3.0. Part 3: Constraints</title>
+    <title>OSLC Change Manazgement Version 3.0. Part 3: Constraints</title>
```
Comment (berezovskyi) [inline specs/cm/change-mgt-shapes.html:5]: With a heavy heart, I must say "Management" wins https://www.oxfordlearnersdictionaries.com/definition/american_english/management 😂

File: specs/cm/change-mgt-shapes.html [hunk 5]
```diff
@@ -2,30 +2,29 @@
...
       content="Specifies the standard constraints on the OSLC Change Management domain vocabulary terms."
     />
 
     <script
-      src="https://cdn.jsdelivr.net/gh/oasis-tcs/tab-respec@v2.1.17/builds/respec-oasis-common.min.js"
+      src="https://cdn.jsdelivr.net/gh/oasis-tcs/tab-respec@v2.1.20/builds/respec-oasis-common.min.js"
       async
       class="remove"
     ></script>
-    <!-- <script src='http://127.0.0.1:9000/respec-oasis-common.js' async class='remove'></script> -->
```
Comment (berezovskyi) [inline specs/cm/change-mgt-shapes.html:17]: This is actually needed to test ReSpec from the master branch... Why did you remove it if I may ask?

**Review by berezovskyi — CHANGES_REQUESTED** [review]
Please see the comments inside. Ignore the QM spec build failure.

File: specs/am/architecture-management-spec.html (hunk 2 above)
Comment (jamsden) [reply specs/am/architecture-management-spec.html:71]: Done

File: specs/am/architecture-management-spec.html (hunk 3 above)
Comment (jamsden) [reply specs/am/architecture-management-spec.html:815]: I'll use https://open-services.net/wiki/architecture-management/OSLC-Architecture-Management-2.0-Appendix-A_-Samples/

File: specs/cm/change-mgt-shapes.html (hunk 5 above)
Comment (jamsden) [reply specs/cm/change-mgt-shapes.html:17]: I thought they were old an no longer needed.

**Review by berezovskyi — APPROVED** [review]

**Comment by berezovskyi** [comment]
For the record, blocked by https://github.com/oasis-open-projects/administration/issues/15

**Comment by berezovskyi** [comment]
@jamsden I think you should also remove CM changes from this PR

**Comment by berezovskyi** [comment]
Also, build errors shall go away if you merge master into the PR branch.

**Comment by jamsden** [comment]
All the lint errors are in ccm-primer. Maybe we should not deliver anything to master that doesn't pass checks so that we don't hold up unrelated pull requests.

**Comment by jamsden** [comment]
@berezovskyi , if this can be merged, then I think we can have OASIS go ahead with the publishing. Do you think we need another ZIP file with the corrections? Or can we let publish the release with the ZIP file OASIS actually used, even though it has errors? That would mean what we tag for the release won't match what's in the ZIPs. But for a PSD, that's probably fine.

File: specs/cm/change-mgt-shapes.html [hunk 6]
```diff
@@ -10,21 +10,22 @@
     />
 
     <script
-      src="https://cdn.jsdelivr.net/gh/oasis-tcs/tab-respec@v2.1.20/builds/respec-oasis-common.min.js"
+      src="https://cdn.jsdelivr.net/gh/oasis-tcs/tab-respec@v2.1.18/builds/respec-oasis-common.min.js"
```
Comment (berezovskyi) [inline specs/cm/change-mgt-shapes.html]: is this intentional?

File: specs/cm/change-mgt-shapes.html [hunk 7]
```diff
@@ -10,21 +10,22 @@
...
-      var oasisBase = "https://docs.oasis-open-projects.org/oslc-op/cm/v3.0/";
+      var oasisBase = "https://docs.oasis-open-projects.org/oslc-op/cm/v3.0";
 
       var specStatus = "PS";
       var revision = "01";
 
       var thisBase = wdBase;
       if (specStatus != "WD") {
-        var thisBase = oasisBase + status.toLowerCase() + revision.toString().padStart(2, "0") + "/";
+        thisBase = oasisBase + "/" + specStatus.toLowerCase() + revision;
```
Comment (berezovskyi) [inline specs/cm/change-mgt-shapes.html]: also, I think it would be better if you touch 1 spec per PR in the future

File: specs/am/architecture-management-shapes.ttl [hunk 8]
```diff
@@ -244,7 +244,7 @@ requirement."""^^rdf:XMLLiteral ;
     oslc:propertyDefinition jazz_am:refine  ;
     oslc:range oslc:Any ;
     oslc:occurs oslc:Zero-or-many;
-    dcterms:description """TThe target is a refinement of the source. (e.g. a use case scenario
+    dcterms:description """The target is a refinement of the source. (e.g. a use case scenario
```
Comment (berezovskyi) [inline specs/am/architecture-management-shapes.ttl:247]: This is an extra fix that is not done in the published files, right?

**Review by berezovskyi — CHANGES_REQUESTED** [review]
Jim, please explain why ReSpec version was reverted in the CM spec file in the PR that is supposed to deal with AM only and then we are (kind of) ready to go with merging this PR in and tagging `am-v3.0-psd01`.

"Kind of" because comparing the changes you did to the ones Paul did nearly got me pulling my hair out! I do appreciate that spec publication became a bit faster but the process became a mess. I also understand we were the ones putting pressure on them. Nevertheless, I propose that going forward OASIS does not make any material changes to the delivered files _except for those needed to produce PDFs._ If there is a typo "TTHe" in the file, we fix it here and produce a new ZIP. Otherwise we risk that the tagged content `am-v3.0-psd01` and what is published on OASIS differ. cc @OASIS-OP-Admin @paul-knight @chet-ensign

**Review by berezovskyi — APPROVED** [review]
